.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: required
+env:
+  global:
+    - FAUNA_ROOT_KEY=secret
+    - FAUNA_DOMAIN=localhost
+    - FAUNA_SCHEME=http
+    - FAUNA_PORT=8443
+jdk:
+  - oraclejdk8
+before_install:
+  - openssl aes-256-cbc -K $encrypted_c0ecaa32cf03_key -iv $encrypted_c0ecaa32cf03_iv -in core.tgz.enc -out core.tgz -d
+  - tar zxf core.tgz
+  - sh install-core.sh
+  - sudo apt-get install -y npm
```

Comment:
Rather, there's no indication that I can see that it's installed so I'm
ensuring its there.

On Thursday, December 17, 2015, Jeff Smick <jeff@faunadb.com> wrote:

> Node is pre-installed in the Travis environment. npm is not.
>
> On Thursday, December 17, 2015, andy-faunadb <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>
>> In .travis.yml
>> <https://github.com/faunadb/faunadb-js/pull/41#discussion_r47983650>:
>>
>> > @@ -0,0 +1,17 @@
>> > +sudo: required
>> > +env:
>> > +  global:
>> > +    - FAUNA_ROOT_KEY=secret
>> > +    - FAUNA_DOMAIN=localhost
>> > +    - FAUNA_SCHEME=http
>> > +    - FAUNA_PORT=8443
>> > +jdk:
>> > +  - oraclejdk8
>> > +before_install:
>> > +  - openssl aes-256-cbc -K $encrypted_c0ecaa32cf03_key -iv $encrypted_c0ecaa32cf03_iv -in core.tgz.enc -out core.tgz -d
>> > +  - tar zxf core.tgz
>> > +  - sh install-core.sh
>> > +  - sudo apt-get install -y npm
>>
>> Usually installing node installs npm. Are you sure you need this?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/faunadb/faunadb-js/pull/41/files#r47983650>.
>>
>